combat-trainer.lic

```diff
@@ -2259,16 +2261,42 @@ class AttackProcess
       game_state.loaded = false
       waitrt?
     elsif game_state.loaded
-      actions = game_state.next_aim_action.split(';')
-      actions.each do |action|
-        first_word = action.split(' ')[0].downcase
-        matches = [/^You /]
-        matches = ['You get', 'You draw out', 'You pick up', "You're already holding that"] if @get_actions.include?(first_word)
-        matches = ['Roundtime',"You aren't close enough to attack"] if @rt_actions.include?(first_word)
-        matches = ['You put', 'You sheathe'] if @stow_actions.include?(first_word)
-        bput(action.strip, matches)
-        pause 0.25
-        waitrt?
+      if game_state.aiming_trainables
+        skill = game_state.determine_aiming_skill
+
+        if skill.eql?('Brawling')
+          3.times{ return if !execute_aiming_action?('gouge', game_state) }
+          return
+        elsif skill.eql?('Tactics')
+          %w[bob weave circle].each do |action|
+            return if !execute_aiming_action?(action, game_state)
+          end
+          return
+        end
+
+        weap = game_state.wield_offhand_weapon(skill)
+        if skill.include?('Thrown')
+          actions = %w[lob lob lob]
+        else
+          actions = %w[jab feint draw]
+        end
+        actions.each do |action|
+          break if !execute_aiming_action?(action, game_state)
+          bput("get my #{weap}", 'You pick up') if action.eql?('lob')
+        end
+        game_state.sheath_offhand_weapon(skill) if left_hand
```

Comment:
💡 You might consider adding a guard clause to `sheath_offhand_weapon` (something like `return unless weapon`) and then you don't need the conditional here

Reply:
Actually, the weapon in sheath-offhand_weapon was misleading, as it takes in a skill and uses the weapon_list[skill] for handling that. We could always have a weapon, but it might have failed to pick it up? I can just remove the condition, either way it's going to be an issue. I also added a conditional in case the brawling/tactics for the previous changes suggested, so that should handle that.